app.js

```diff
@@ -1,157 +1,209 @@
 'use strict';
 
 var correctAnswers = [];
-var wrongAnswers = [];
+// var wrongAnswers = [];
```

Comment:
Let's get rid of this commented out code. No need for it if it's not being used.